src/main/webapp/styles.css

```diff
@@ -1,176 +1,181 @@
 /* Overriding default styles */
 * {
-    box-sizing: border-box;
+  box-sizing: border-box;
 }
 
 body {
-    margin: 0;
-    background: #333;
-    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  margin: 0;
+  background: #333;
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
```

Comment:
There is inconsistency in css file code style. There us tabs used in another PR and 2 spaces used here. Please try to use common code styles amongst yourselves